src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/CreateSaveSearch/SaveSearchCommand.cs

```diff
@@ -2,16 +2,16 @@
 using System.Collections.Generic;
 using MediatR;
 
-namespace SFA.DAS.FindAnApprenticeship.Application.Commands.SaveSearch;
+namespace SFA.DAS.FindAnApprenticeship.Application.Commands.SaveSearch.CreateSaveSearch;
 
 public record SaveSearchCommand(
     Guid Id,
     Guid CandidateId,
     bool DisabilityConfident,
     decimal? Distance,
-    string? Location,
-    string? SearchTerm,
-    List<int>? SelectedLevelIds,
-    List<int>? SelectedRouteIds,
+    string Location,
```

Comment:
Is there a reason to make these non-nullable? This could introduce bugs unless every representation of the search parameters is non-nullable.